lib/Service/Appointments/AvailabilityGenerator.php

```diff
@@ -79,11 +80,17 @@ public function generate(AppointmentConfig $config,
 
 		// If we reach this state then there are no available dates anymore
 		if ($latestEnd <= $earliestStart) {
+			$this->logger->debug('Appointment config ' . $config->getToken() . ' has {latestEnd} as latest end but {earliestStart} as earliest start. No slots available.', [
+				'latestEnd' =>  $latestEnd,
+				'earliestStart' => $earliestStart,
+				'app' => 'appointments'
```

Comment:
Would still prefix it like the following to avoid confusion with https://apps.nextcloud.com/apps/appointments

```suggestion
				'app' => 'calendar-appointments'
```

Reply:
Good point.